hal/src/main/python/semiwrap/DriverStationTypes.yml

```diff
@@ -12,11 +12,12 @@ enums:
   HAL_MatchType:
     value_prefix: HAL_kMatchType
   HAL_RobotMode:
-    ignore: true
   RobotMode:
+    rename: _RobotMode
```

Comment:
I didn't know if it is better to rename this enum class or the `HAL_RobotMode` one so I randomly picked. Or if there is a better thing to rename them too (same for ControlWord